gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "1.4.3",
+  "version": "1.4.4",
```

Comment:
Hmm, technically this is a breaking change in how the footers are supposed to be structured.

Reply:
> Hmm, technically this is a breaking change in how the footers are supposed to be structured.

As discussed I have created a feature branch for V2 breaking changes